pilot/pkg/serviceregistry/kube/controller/controller_test.go

```diff
@@ -556,6 +563,7 @@ func TestGetProxyServiceInstances(t *testing.T) {
 				ServicePort: &model.Port{Name: "tcp-port", Port: 8080, Protocol: protocol.TCP},
 				Endpoint: &model.IstioEndpoint{
 					Address:         "129.0.0.2",
+					Network:         networkID,
```

Comment:
Can we also test that when a Pod already has the network label set, it gets preserved (rather than using the controller-default)

Reply:
Probably easier to do so in `TestNewEndpointBuilderTopologyLabels`

Reply:
I have added cases there